soc/nordic_nrf/nrf51/Kconfig

```diff
@@ -7,10 +7,7 @@
 config SOC_SERIES_NRF51X
 	select ARM
 	select CPU_CORTEX_M0
-	select SOC_FAMILY_NRF
 	imply XIP
 	select HAS_NRFX
 	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
 	select HAS_POWEROFF
-	help
-	  Enable support for NRF51 MCU series
```

Comment:
why does 87c82c111c46bda97607048e8a320b7bdf950897 contain changes to other soc series than 54?

Reply:
Because the original port for nordic was not done properly and this was needed to actually port it properly

Reply:
then please, split commits, amend original commits, change message or anything else that makes things clear.

Reply:
Commit message updated